vite_rails/lib/vite_rails/tag_helpers.rb

```diff
@@ -88,10 +88,15 @@ def vite_manifest
 
   # Internal: Renders a modulepreload link tag.
   def vite_preload_tag(*sources, crossorigin:, **options)
-    sources.map { |source|
-      href = path_to_asset(source)
-      try(:request).try(:send_early_hints, 'Link' => %(<#{ href }>; rel=modulepreload; as=script; crossorigin=#{ crossorigin }))
+    asset_paths = sources.map { | source | path_to_asset(source) }
+    try(:request).try(
+      :send_early_hints,
+      'Link' => asset_paths.map do |href|
+        %(<#{ href }>; rel=modulepreload; as=script; crossorigin=#{ crossorigin })
+      end.join(", "),
```

Comment:
Should the separator be a [new line](https://github.com/puma/puma/blob/ce5bd84990a6abb99c04ccc620da9a429da54e8a/lib/puma/request.rb#L539C25-L539C32) here as well?

```suggestion
      end.join("\n"),
```

Reply:
Fixed
Forgot where I read comma as separator...